frontend/components/ifixit-image/ifixit-image.tsx

```diff
@@ -66,12 +66,28 @@ function isStrapiImage(src: string) {
    );
 }
 
-function getImageLoader(sizeMap: SizeMap, defaultSize: string): ImageLoader {
+function getImageLoader(
+   sizeMap: SizeMap,
+   defaultSize: string,
+   baseWidth?: number | string
+): ImageLoader {
    return ({ src, width }: ImageLoaderProps) => {
       const baseSrc = src.replace(/\.[^/.]+$/, '');
-      const sizeName = getImageSize(width, sizeMap, defaultSize);
+      let realWidth = width;
+      if (typeof baseWidth === 'string') {
+         realWidth = parseInt(baseWidth);
+      } else if (baseWidth) {
+         realWidth = baseWidth;
+      }
+      // If next wants to see the 2x image here then this accounts for that in our calculations of resolution
+      // see next's image.js getWidths() for more details
+      if (width / realWidth > 2) {
+         realWidth *= 2;
+      }
```

Comment:
Consider avoiding the continued mutation of `realWidth` by making this a new constant:
```suggestion
      const scaledWidth = (width / realWidth > 2) ? realWidth * 2 : realWidth;
```

Also, I'm wondering if this needs to be
```js
width / realWidth >= 2
```
because if we expect `width` to be 2x `realWidth`, I think the `>` version won't actually get correct answers.

Reply:
So it needs to be `> 2` because if its  `= 2` next will just handle it in the backend. This case is basically supposed to handle if we for some reason  have an image size between `width * 2` and `realwidth * 2`, then we should scale to the minimal one.  However, after syncing the sizes in 880f96a8d9a051e5aaf4de131165fae9b49cc0c1 im not sure how likely that case is to run into (it happened more when we could scale up to our cart image size then `next` would scale _that_ up to its next size). So i actually need to think over if this change is necessary at all